src/main/res/values-v21/themes.xml

```diff
@@ -1,55 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Similarly, minsdkversion is 21.

Reply:
I merged both themes.xml (v21) and themes.xml (base), then since explicit v21 qualifier (because minsdkversion is 21 ) is not required I removed themes.xml (v21) file.